File: exercicios/moj/contest-1/b/b.py
N = int(input())
ans = [float(a) * i for i, a in enumerate(input().split(), 1)]
tot = sum(ans) / ((len(ans) / 2) * (1 + len(ans)))
print(f"Sim\n{format(tot, '.2f')}" if float(format(tot, '.2f')) >= 5.0 else f"Nao\n{format(tot, '.2f')}")

File: exercicios/moj/contest-1/b/b.cpp
#include <bits/stdc++.h>

using namespace std;

int main(void) {
  ios::sync_with_stdio(false);
  cin.tie(NULL);

  int N;
  cin >> N;

  unsigned long sum = 0;
  double T, grade = 0;

  for (int i = 1; i <= N; i++) {
    cin >> T;
    sum += i;
    grade = grade + (T * i);
  }

     
  cout << ((grade / sum) >= 5.0 ? "Sim" : "Nao") << '\n';
  cout << setprecision(3) << (grade / sum) << '\n';

  return 0;
}